hack/run-locally.sh

```diff
@@ -180,6 +191,7 @@ The following options are always accepted but only used for new clusters:
  -f               the path to an openshift-install-created install-config.yaml file; if not given one will be created
  -i               path to the openshift-install binary; if not given PATH will be searched
  -n [net plugin]  the name of the network plugin to deploy; one of [sdn|OpenShiftSDN|ovn|OVNKubernetes]
+ -s [secret]      additional pull secret to add to install-config.yaml
```

Comment:
What's the use case here? Wouldn't it be better to just add that pull secret to your template install-config permanently?